### DISCUSSION TITLE
Troubleshooting Information - missing markdown?

### DISCUSSION THREAD

**Discussion by u-RraaLL** [uBlockOrigin/uBlock-issues#1880] [Q&A] [answered]
Didn't early iterations of the Troubleshooting Information in the Support pane use markdown (lines were prepended with 4 spaces)?

I know the report buttons use triple backtics to codeblock instead, but I think "space" code-blocking should still be available for people manually copying their config. Isn't that what the [Select all] button is for?

**Comment by gorhill** [comment, marked as answer]
I removed the 4-space prefix because most of the time hat information is uploaded through a URL parameter, and in such case not having to encode 4 extra spaces per line saves on URL length. For the occasional case of copy-paste, there is nothing in there which requires escaping, so all should still render fine even if not rendered as code.

**Reply by u-RraaLL** [reply]
If I'm not mistaken, reddit will render the config as a single line... It's not _that_ much work to fix it on desktops, but trying to do the same on mobiles might be a quite a chore.

Any way to display/select it prepended and disregard the first 4 spaces when using the report/file issue buttons?
